update-affiliated/update_astropy_helpers.py

```diff
@@ -111,21 +118,23 @@ def open_pull_request(fork, repo):
     run_command('git add astropy_helpers')
     run_command('git add ah_bootstrap.py')
     run_command('git add ez_setup.py')
-    run_command('git commit -m "Updated astropy-helpers to {0}"'.format(HELPERS_TAG))
+    run_command(
+        'git commit -m "Updated astropy-helpers to {0}"'.format(HELPERS_TAG))
 
     run_command('git push origin {0}'.format(BRANCH))
 
     print(tmpdir)
 
     report_user = '@astrofrog'
 
-    if username != 'astrofrog':
-        report_user = '@{} or @astrofrog'.format(username)
+    if user.login != 'astrofrog':
+        report_user = '@{} or @astrofrog'.format(user.login)
 
-    repo.create_pull(title='Update astropy-helpers to {0}'.format(HELPERS_TAG),
-                     body=HELPERS_UPDATE_MESSAGE_BODY.format(HELPERS_TAG, report_user),
-                     base='master',
-                     head='{0}:{1}'.format(fork.owner.login, BRANCH))
+    repo.create_pull(
+        title='Update astropy-helpers to {0}'.format(HELPERS_TAG),
+        base='master',
```

Comment:
Note: `create_pull` now has positional arguments, so I rearranged them, just to be on the safe side.